docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+# Please refer https://aka.ms/HTTPSinContainer on how to setup an https developer certificate for your ASP.NET Core service.
+
+version: '3.9'
+
+services:
+  db:
+    image: mcr.microsoft.com/mssql/server:2019-latest
+    environment:
+      ACCEPT_EULA: 'Y'
+      MSSQL_SA_PASSWORD_FILE: /run/secrets/db_password
+    ports:
+      - 1433:1433
+    secrets:
+      - db_password
+
+  # storage:
```

Comment:
Dette bruger ondFrisk. Jeg ved ikke om vi får brug for det på et tidspunkt. Kan eventuelt slettes.

Reply:
Tænker vi bare lader det stå indtil videre så!